gpumon.opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "John Else" ]
+homepage: "https://github.com/xenserver/gpumon"
+bug-reports: "https://github.com/xenserver/gpumon/issues"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "base-threads"     
+  "ounit"
```

Comment:
should this have an annotation that we require this only during tests?

Reply:
Good idea.

Reply:
Regarding Travis: maybe we could use a mock implementation when we run Travis?

Reply:
We could define a separate .c file for Travis with the mock implementation (or supply a mock header file, whichever is easier)